core/templates/base-components/base-content.component.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the Base Transclusion Component.
+ */
+
+import { Component, Input } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import constants from 'assets/constants';
```

Comment:
Use `app.constants.ts` which also contains fields from constants.ts. 

cc @DubeySandeep @srijanreddy98  Should we introduce a lint check for this?

Reply:
done

Reply:
@vojtechjelinek @srijanreddy98 Feel free to create an issue with the requirement, I'll work with linter team to triage the issue. Thanks!

Reply:
Done #12910